.gitignore

```diff
@@ -32,5 +32,6 @@ coverage/
 !.yarn/plugins
 !.yarn/releases
 !.yarn/sdks
+!.yarn/versions
```

Comment:
Do we need this to be checked in, e.g., there's a version conflict?

Reply:
I think we can safely remove it - I added it for testing some hypothesis and forgot to remove it.
We can proceed without that